x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -55,33 +54,44 @@ export const LatestFindingsContainer = ({ dataView }: { dataView: DataView }) =>
         setQuery={setUrlQuery}
         query={urlQuery.query}
         filters={urlQuery.filters}
-        loading={findingsGroupByNone.isLoading}
+        loading={findingsGroupByNone.isFetching}
       />
       <PageWrapper>
-        <PageTitle>
-          <PageTitleText
-            title={
-              <FormattedMessage id="xpack.csp.findings.findingsTitle" defaultMessage="Findings" />
-            }
-          />
-        </PageTitle>
+        <LatestFindingsPageTitle />
         <FindingsGroupBySelector type="default" />
         <FindingsDistributionBar
           total={findingsGroupByNone.data?.total || 0}
           passed={findingsCount.data?.passed || 0}
           failed={findingsCount.data?.failed || 0}
-          pageStart={urlQuery.from + 1} // API index is 0, but UI is 1
-          pageEnd={urlQuery.from + urlQuery.size}
+          pageStart={urlQuery.pageIndex * urlQuery.pageSize + 1} // API index is 0, but UI is 1
+          pageEnd={urlQuery.pageIndex * urlQuery.pageSize + urlQuery.pageSize}
```

Comment:
since the query interface changed, the calculation changed too. 

but we already know it's [wrong](https://github.com/elastic/cloudbeat/issues/89) and the quick fix is roughly: 

```diff
+ pageStart={urlQuery.pageIndex * urlQuery.pageSize + 1}
+ pageEnd={urlQuery.pageIndex * urlQuery.pageSize + findingsGroupByNone.data.page.length}
```

which takes into account the actual page size for the `pageEnd`, and doesn't assume all buckets are the same size. 

maybe we can include it here


@kfirpeled